gst-libs/mfx/gstmfxallocator_d3d11.c

```diff
@@ -49,7 +49,7 @@ gst_mfx_task_frame_alloc(mfxHDL pthis, mfxFrameAllocRequest * request,
   if (task->task_type & (GST_MFX_TASK_VPP_OUT | GST_MFX_TASK_DECODER))
   request->Type |= 0x1000;*/
 
-  if (priv->saved_responses && priv->task_type & GST_MFX_TASK_DECODER) {
+  if (priv->saved_responses && (priv->task_type & GST_MFX_TASK_DECODER) == GST_MFX_TASK_DECODER) {
```

Comment:
this change is just cosmetic, you can revert it without changing the result in any way since all the task types are different powers of 2. I did it while cleaning up some parts before debugging but I didn't pay attention to this whole block. Is its purpose to avoid allocating several times for the decoder, or something else?